pages/juang961225/style.css

```diff
@@ -0,0 +1,115 @@
+body {
+    display: flex;
+    flex-wrap: nowrap;
+    margin: 0px;
+    padding: 0px;
+}
+.main {
+    align-items: flex-start;
+    background-image: url(../juang961225/pictures/FOR\ THE\ HORDA.jpg);
+    background-position: center;
+    background-size: contain;
+    display: flex;
+    display: flex;
+    flex-wrap: wrap;
+    height: 100%;
+    justify-content: center;
+    position: absolute;
+    width: 100%;
+}
+
+.main__title {
+    color: white;
+    display: initial;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 300%;
+    margin:3%;
+    text-transform: uppercase;
+}
+
+.main__container {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    height: 60%;
+    justify-content: center;
+    margin: 0%;
+    padding: 0%;
+    width: 100%;
+}
+
+.main__button {
+    align-items: center;
+    background: transparent;
+    border-radius: 100%;
+    border: 1.5% solid #ffffff;
+    display: flex;
+    height: 9%;
+    justify-content: center;
+    width: 3%;
+}
+
+.main__container-button {
+    color: white;
+}
+
+.main__card {
+    background:rgb(92, 91, 91);
+    border-radius: 3%;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    height: 100%;
+    margin: 2%;
+    width: 15%;
+}
+
+.main__card:hover {
+    background: rgba(179, 179, 179, 0.479);
+}
+
+.main__img {
+    border-radius: 3% 3% 0% 0%;
+    height: 50%;
+    width: 100%;
+    
+}
+
+.main__card-title {
+    color: rgb(167, 12, 12);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size:100%;
+    margin: 5% 2% 2% 0% ;
+    padding: 0% 10% 0% 3%;
+    text-transform: uppercase;
+}
+
+.main__content {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 80%;
+    margin: 0% 5% 0% 5%;
```

Comment:
For the general CSS file, I think you should reconsider using percentages as general units of measures, instead, I suggest using px's or even relative units if you know how to work with them. if you want to use percentages you should use these units for the width of the containers (Most pretty common use).

Reply:
solved

Reply:
okay, thanks for the advice